.github/workflows/lint.yml

```diff
@@ -17,21 +16,21 @@ jobs:
           version: v1.43.0
           args: --timeout 5m0s
 
-  md-link-check:
-    name: Check Markdown links
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Check if Markdown links are valid
-        uses: gaurav-nelson/github-action-markdown-link-check@v1
-        with: 
-          config-file: '.github/linters/mlc_config.json'
-          folder-path: "."
+#  md-link-check:
```

Comment:
We can't leave it commented

Reply:
Oh, and we should solve this before merging

Reply:
As I can see, for now links to images are working. I'll get back this check